girder/utility/install.py

```diff
@@ -148,6 +155,9 @@ def main():
     plugin.set_defaults(func=install_plugin)
     plugin.add_argument('-f', '--force', action='store_true',
                         help='Overwrite plugins if they already exist.')
+    plugin.add_argument('-e', '--editable', action='store_true',
+                        help='Install by symlinking to the plugin directory.')
```

Comment:
I'm in favor of sticking closer to ``pip install``'s conventions, but I see nothing wrong with also adding a ``--symlink`` flag as a documented alias.